src/routes/session/session-get.ts

```diff
@@ -2,6 +2,14 @@ import { Request, Response } from "express";
 import { createUrlWithQueryString } from "utils/create-url-with-query-string";
 
 export const SessionGet = (req: Request, res: Response) => {
+	if (req.query.resetSession) {
+		const cookies = req.cookies;
+		for (const cookieName in cookies) {
+			res.clearCookie(cookieName);
```

Comment:
I think you are doing right that also clear the jiraHost as well inside the session, so that multi tab on multi jira host can work perfectly.
However, I can think of one edge case that current support but this pr does not.

At the moment, if user stay in the jira-configuration page for a long time (and give github token already present in cookie), they can setup new org in this new page without problem.

But in the PR, since jiraHost is cleared, when they click the link, there will be no new jwt (from `AP.requestToken()`) passed to the backend, and backend will not found a jiraHost (before doing oauth dance).
Previously, since jiraHost is NOT cleared, backend will proceed to oauth dance.

However, I don't think the fault lies in this PR, it lies in our standalone page weirdness. We should revisit our standalone page.